src/Native/Runtime/wasm/AsmOffsetsCpu.h

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//
+// This file is used by AsmOffsets.h to validate that our
+// assembly-code offsets always match their C++ counterparts.
+//
+// NOTE: the offsets MUST be in hex notation WITHOUT the 0x prefix
+
+PLAT_ASM_SIZEOF(A0, ExInfo)
+PLAT_ASM_OFFSET(0, ExInfo, m_pPrevExInfo)
+PLAT_ASM_OFFSET(4, ExInfo, m_pExContext)
+PLAT_ASM_OFFSET(8, ExInfo, m_exception)
+PLAT_ASM_OFFSET(0c, ExInfo, m_kind)
+PLAT_ASM_OFFSET(0d, ExInfo, m_passNumber)
+PLAT_ASM_OFFSET(10, ExInfo, m_idxCurClause)
+PLAT_ASM_OFFSET(14, ExInfo, m_frameIter)
+PLAT_ASM_OFFSET(9C, ExInfo, m_notifyDebuggerSP)
+
+PLAT_ASM_OFFSET(0, PInvokeTransitionFrame, m_RIP)
```

Comment:
PInvokeTransitionFrame should not be needed for WASM here.

Reply:
Done